examples/ota-provider-app/linux/main.cpp

```diff
@@ -147,6 +149,9 @@ bool HandleOptions(const char * aProgram, OptionSet * aOptions, int aIdentifier,
             gOtaFilepath = aValue;
         }
         break;
+    case kOptionImageUri:
```

Comment:
Note that if the supplied URI is longer than 256 bytes then it will be silently truncated.

Reply:
Thanks for comment. This is mostly for testers to use to test invalid image URIs so not likely to surpass 256 bytes. For most cases where we want normal/happy path, this command line option *should not* be supplied.